src/base/bittorrent/abstractfilestorage.cpp

```diff
@@ -90,6 +93,6 @@ void BitTorrent::AbstractFileStorage::renameFolder(const Path &oldFolderPath, co
     for (const int index : renamingFileIndexes)
     {
         const Path newFilePath = newFolderPath / oldFolderPath.relativePathOf(filePath(index));
-        renameFile(index, newFilePath);
+        renameFile(index, newFilePath); // note; if file would be overwritten, we note that is true here
```

Comment:
The comment audience is unclear and doesn't bear any context for the developer. Just don't add it.
```suggestion
        renameFile(index, newFilePath);
```